tests/test_api_service.py

```diff
@@ -1,37 +1,53 @@
+import pytest
 from fastapi.testclient import TestClient
 
-from comprehensive_microservice.api_service import app
+from comprehensive_microservice.api_service import app, get_storage
+from comprehensive_microservice.coder import Storage
 
-tester = TestClient(app)
 
+@pytest.fixture()
+def storage():
+    return Storage()
 
-def test_correct_data():
-    data_json = {"text": "some text", "title": "some title"}
-    response = tester.post("/save", json=data_json)
+
+@pytest.fixture()
+def client(storage):
+    app.dependency_overrides[get_storage] = lambda: storage
```

Comment:
overriding dependencies for tests (needed for future when we implement DB storage)